gbpservice/neutron/db/migration/alembic_migrations/versions/d2aab79622fe_nfp_enterprise_db.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2016 OpenStack Foundation
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+# revision identifiers, used by Alembic.
+revision = 'd2aab79622fe'
+down_revision = 'c1aab79622fe'
```

Comment:
This is a hard-coded value, it wont be same always, it'll get change with gbp opensource code.
We need to maintain this somehow or whenever we are shifting from advanced to enterprise, always get the latest head, and fill that value as down revision.